kani-compiler/src/codegen_cprover_gotoc/context/goto_ctx.rs

```diff
@@ -59,6 +59,7 @@ pub struct GotocCtx<'tcx> {
     pub current_fn: Option<CurrentFnCtx<'tcx>>,
     pub type_map: FxHashMap<InternedString, Ty<'tcx>>,
     pub proof_harnesses: Vec<HarnessMetadata>,
+    pub test_harnesses: Vec<HarnessMetadata>,
```

Comment:
What is a test-harness? Is that anything with the `test` attribute?

Reply:
Yep